src/markdown-pages/install-with-kurl/managing-nodes.md

```diff
@@ -261,3 +274,32 @@ To use the EKCO add-on to remove a node:
    The EKCO purge script For information about the EKCO purge script, see [Purge Nodes](/docs/add-ons/ekco#purge-nodes) in _EKCO Add-on_.
 
 1. Remove the node from the cluster.
+
+## Troubleshoot Node Removal
+
+This section includes information about troubleshooting issues with node removal in Rook Ceph clusters.
+
+### Rook Ceph v1.0.4 is Unhealthy with Mon Pods Not Rescheduled
+
+#### Symptom
+
+When you run `kubectl -n rook-ceph exec deployment.apps/rook-ceph-operator -- ceph status` for a Rook Ceph v1.0.4 cluster, you see that Ceph is in an unhealthy state where failed mons are not rescheduled. 
+
+For example:
+
+```
+health: HEALTH_WARN
+        1/3 mons down, quorum a,c
+```
+
+#### Cause
+
+This is caused by a known issue in Rook Ceph v1.0.4 where failed mons are not removed during mon failover. For more information, see [Mon is never rescheduled](https://github.com/rook/rook/issues/2262#issuecomment-460898915) in the rook GitHub repository.
+
+#### Solution
+
+To resolve this issue, manually stop the Rook Ceph operator, remove the failed mon from the configmap, delete the failed mon, and then rescale the operator.  
+
```

Comment:
Please, ensure that after the changes made above all mon pods will be running and the Ceph status is healthy.

Reply:
I think we need to add the step to verify that the fix was done right.

Reply:
I added two steps to the end of the procedure to check that the mon pods are all running and to check that ceph is healthy